src/test/java/software/amazon/awssdk/crt/test/S3ClientTest.java

```diff
@@ -1453,6 +1395,11 @@ public void testS3Copy() {
         skipIfNetworkUnavailable();
         Assume.assumeTrue(hasAwsCredentials());
 
+        String COPY_SOURCE_BUCKET = "aws-crt-test-stuff-us-west-2";
```

Comment:
this is still hard-coded though?

Reply:
https://github.com/awslabs/aws-crt-java/pull/788/files#diff-61fd3ff6b0c1bb48020cafcdd8ae2a00f7ba2e2da6aff15d125868eda22bdc96R1390-R1393

The copy tests are disabled